src/lib/components/scrollbarwrapper/ScrollbarWrapper.component.js

```diff
@@ -25,12 +26,15 @@ const ScrollbarContainer = styled.div`
 
         ::-webkit-scrollbar-thumb {
           width: 4px;
+          height: 4px;
           background: ${brand.border};
           border-radius: 4px;
           -webkit-border-radius: 4px;
           background-clip: padding-box;
           border-left: 2px solid rgba(0, 0, 0, 0);
           border-right: 2px solid rgba(0, 0, 0, 0);
+          border-top: 2px solid rgba(0, 0, 0, 0);
+          border-bottom: 2px solid rgba(0, 0, 0, 0);
```

Comment:
Would be better to merge them into one line.
And instead of `rgba(0, 0, 0, 0)` shouldn't we use the predefined color from the theme?

Reply:
Absolutely ^^ sorry about that. Done